test/ensureArray.js

```diff
@@ -0,0 +1,29 @@
+import chai from 'chai';
+import * as RA from '../src/index';
+import eq from './shared/eq';
+
+
+describe('ensureArray', function () {
+  it('tests `Array` values', function () {
+    const emptyArray = [];
+    const filledArray = [42];
+    const arrayInstance = new Array();
+
+    chai.assert.strictEqual(RA.ensureArray(emptyArray), emptyArray);
+    chai.assert.strictEqual(RA.ensureArray(filledArray), filledArray);
+    chai.assert.strictEqual(RA.ensureArray(arrayInstance), arrayInstance);
+    chai.assert.strictEqual(RA.ensureArray(Array.prototype), Array.prototype);
+  });
+
+  it('tests other values', function () {
```

Comment:
`it('returns non-Array values as the only element of a new Array', function () {`